src/qt/askpassphrasedialog.cpp

```diff
@@ -234,6 +235,15 @@ bool AskPassphraseDialog::event(QEvent *event)
     return QWidget::event(event);
 }
 
+void AskPassphraseDialog::setShowPass(bool showPass)
```

Comment:
Nit, this is not a setter, just `void showPassword(bool show)`.

Reply:
Changed.  Thanks for the pointer, I'm new to Qt.